app_dart/lib/src/request_handlers/get_status.dart

```diff
@@ -30,15 +31,36 @@ class GetStatus extends RequestHandler<Body> {
   final DatastoreServiceProvider datastoreProvider;
   final BuildStatusProvider buildStatusProvider;
 
+  static const String ownerKeyParam = 'ownerKey';
```

Comment:
I am unclear of what this does. It could be more descriptive if called `commitKey`, but I still don't know what it's for.

Reply:
Maybe `lastCommitKey`?

Reply:
`lastCommitKey` works!